app/models/consultation.rb

```diff
@@ -105,22 +105,22 @@ def send_consultee_emails(attributes)
   end
 
   def start_deadline
-    update!(end_date: end_date_from_now, start_date: start_date || 1.business_day.from_now)
+    update!(end_date: end_date_from_now, start_date: start_date || default_start_date)
   end
 
   def extend_deadline(new_date)
-    update!(end_date: [end_date, new_date].max)
+    update!(end_date: [end_date, new_date.end_of_day].max)
```

Comment:
I wonder if some of these date times would be better as dates, and then treat the time of day as implicit? I don't know if that's even possible or if we'd always just get a date time back from postgres.